media/video/__init__.py

```diff
@@ -0,0 +1,80 @@
+"""Upload module"""
+import logging
+from bson import ObjectId
+from flask import request
+import json
+from media import get_collection, blueprint
+from flask import Blueprint
+
+bp = Blueprint('project', __name__)
+
+
+class JSONEncoder(json.JSONEncoder):
+    def default(self, o):
+        if isinstance(o, ObjectId):
+            return str(o)
+        return json.JSONEncoder.default(self, o)
+
+
+@bp.route('/project', methods=['POST'])
```

Comment:
Later we can better organise files. For example we can put routes into routes.py in the app folder. 
No need to do it now.